rebasehelper/application.py

```diff
@@ -777,6 +777,17 @@ def get_rebasehelper_data(self):
         rh_stuff['logs'] = self.get_all_log_files()
         return rh_stuff
 
+    def apply_changes(self):
+        repo = git.Repo(self.execution_dir)
```

Comment:
This could raise an exception if there is no repository. You should initialize one in such case.